fizzbuzz/C/IsabelleFizzBuzz/fizzbuzzpingpongSolution.h

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include <stdlib.h>
+
+
+char* fizztest(int n){
+
+	char str[50] = "";
+
+	if((n% 3 == 0) || (n%5 == 0) || (n%7 == 0) || (n%11 == 0)) {
+
+		if(n%3 == 0){
+			strcat(str, "Fizz");			
+		}
+		if(n%5 == 0){
+			strcat(str, "Buzz");
+		}
+		if(n %7 == 0){
+			strcat(str, "Ping");
+		}
+		if(n%11 == 0){
+			strcat(str, "Pong");
+		}
+
+	}
+
+	if((n% 3 != 0) && (n%5 != 0) && (n%7 != 0) && (n%11 != 0)) 
+	{
+	//copy n into str
+		char string[50] = "";
+		char buffer[10];
+		sprintf(buffer, "%d", n);
+		strcat(str, buffer);
+	}
```

Comment:
one idea... you could use the string buffer from above to determine if any of the conditions were hit before.  i.e. if str != ""  This could also remove the need to check multiple times for if the number is a multiple (line 11 could go away as well)